### DISCUSSION TITLE
Router tab & Mixer lightning first integration - v.1.0.0-alpha.2

### DISCUSSION THREAD

**Discussion by AdelaideSky** [Adelenade/GoXlr-Macos#16] [General]
Nothing more to say, also added bundled daemon.

Now you have an installer, so no need to install the old version !

PLEASE NOTE that you'll need to delete ALL the aggregate device (chat, system etc) in Audio Midi Setup in order to successfully pass the audio setup !

<hr /><em>This discussion was created from the release <a href='https://github.com/Adelenade/GoXlr-Macos/releases/tag/v1.0.0-alpha.2'>Router tab & Mixer lightning first integration - v.1.0.0-alpha.2</a>.</em>

**Comment by mcewball13** [comment]
Is this version working with other MacOS versions or just ventura?

**Reply by AdelaideSky** [reply]
Starting from the first alpha of v1, all the versions will only be on macOS Ventura and newer ! As I use new functions of the macOS 13 SDK...

**Comment by mcewball13** [comment]
Ok so monetary is supported in the alpha?

Sent from my iPhone

On Aug 19, 2022, at 7:46 AM, Adélaïde Sky ***@***.***> wrote:

﻿

Starting from the first alpha of v1, all the versions will only be on macOS Ventura and earlier ! As I use new functions of the macOS 13 SDK...

**Reply by AdelaideSky** [reply]
Oh no sorry I wanted to say "on Ventura and newer versions of macOS" but my English is terrible 🫠

So nope and it's not going to be supported ! You can wait the launch of the stable version ( mid September if I remember correctly ) or install the public beta in order to use the newer versions of the app !
